pkg/skaffold/deploy/kpt_test.go

```diff
@@ -49,18 +49,124 @@ func TestKpt_Deploy(t *testing.T) {
 
 func TestKpt_Dependencies(t *testing.T) {
 	tests := []struct {
-		description string
-		expected    []string
-		shouldErr   bool
+		description    string
+		cfg            *latest.KptDeploy
+		createFiles    map[string]string
+		kustomizations map[string]string
+		expected       []string
+		shouldErr      bool
 	}{
 		{
-			description: "nil",
+			description: "bad dir",
+			cfg: &latest.KptDeploy{
+				Dir: "invalid_path",
+			},
+			shouldErr: true,
+		},
+		{
+			description: "empty dir and unspecified fnPath",
+			cfg: &latest.KptDeploy{
+				Dir: ".",
+			},
+		},
+		{
+			description: "dir",
+			cfg: &latest.KptDeploy{
+				Dir: ".",
+			},
+			createFiles: map[string]string{
+				"foo.yaml":  "",
+				"README.md": "",
+			},
+			expected: []string{"foo.yaml"},
+		},
+		{
+			description: "dir with subdirs",
+			cfg: &latest.KptDeploy{
+				Dir: ".",
+			},
+			createFiles: map[string]string{
+				"food.yml":          "",
+				"foo/bar.yaml":      "",
+				"foo/bat/bad.yml":   "",
+				"foo/bat/README.md": "",
```

Comment:
suggest adding a path containing the "\\"

Reply:
Good idea. I will also add a path with `\`.